cloudtools/aws/ami.py

```diff
@@ -12,28 +12,29 @@
 
 
 def ami_cleanup(mount_point, distro, remove_extra=None):
-    remove_extra = remove_extra or []
-    remove = [
-        "root/*.sh",
-        "root/*.log",
-        "root/userdata",
-        "var/lib/puppet",
-        "etc/init.d/puppet"
-    ]
-    with cd(mount_point):
-        for e in remove + remove_extra:
-            run('rm -rf %s' % (e,))
-        run("sed -i -e 's/127.0.0.1.*/127.0.0.1 localhost/g' etc/hosts")
-        put("%s/fake_puppet.sh" % AMI_CONFIGS_DIR,
-            "usr/sbin/fake_puppet.sh", mirror_local_mode=True)
-        # replace puppet init with our script
-        if distro == "ubuntu":
-            put("%s/fake_puppet.conf" % AMI_CONFIGS_DIR,
-                "etc/init/puppet.conf", mirror_local_mode=True)
-            run("echo localhost > etc/hostname")
-        else:
-            run("ln -sf /usr/sbin/fake_puppet.sh etc/init.d/puppet")
-            run('echo "NETWORKING=yes" > etc/sysconfig/network')
+    if distro != "win2008":
```

Comment:
I went with the second option (https://github.com/mozilla/build-cloud-tools/commit/e19cac64c77c668257a483871bcf2450e30a0ed0), since we'd still need if/else in the code and I'd like to minimise the change/churn in this PR.